src/types/Comment.tsx

```diff
@@ -0,0 +1,7 @@
+export interface Comment {
```

Comment:
you can set all types in this file `react-app-env.d.ts`, and then no need to import them